lineman/app/components/thread_page/proposal_positions_panel/proposal_positions_panel.coffee

```diff
@@ -6,7 +6,8 @@ angular.module('loomioApp').directive 'proposalPositionsPanel', ->
   controller: ($scope, Records, CurrentUser, ModalService, VoteForm) ->
 
     $scope.changeVote = ->
-      ModalService.open VoteForm, proposal: -> $scope.proposal
+      ModalService.open VoteForm, 
+        proposal: -> $scope.proposal.clone()
```

Comment:
why clone proposal here? (we operate on the vote) Does no harm I can see though.